### DISCUSSION TITLE
Sheet presentations

### DISCUSSION THREAD

**Discussion by Cinkowski** [pointfreeco/swift-composable-architecture#2510] [General]
Hi, 
I am wonder if I can change sheet height which is presented by state.routes.presentSheet, which is provided by Reducer?

**Comment by mbrandonw** [comment]
Hi @Cinkowski, sheet heights are controlled by the [`detents` view modifier](https://developer.apple.com/documentation/uikit/uisheetpresentationcontroller/3801903-detents). You can use it in the exact same way you would use it in vanilla SwiftUI.
